src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Security/CSharpDataSetDataTableInIFormatterSerializableObjectGraphAnalyzer.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.NetCore.Analyzers.Security;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public sealed class CSharpDataSetDataTableInIFormatterSerializableObjectGraphAnalyzer
+        : DataSetDataTableInIFormatterSerializableObjectGraphAnalyzer
+    {
+        protected override string ToString(TypedConstant typedConstant)
```

Comment:
I went for the abstract method but I could also pass the function to the ctor, let me know your preference.